tb/cocotb/soc/test_spi.py

```
@@ -9,15 +9,15 @@
 #   0x08  SPI_STATUS  RO        [0]=busy [1]=tx_full [2]=tx_empty
-#                               [3]=rx_empty [4]=rx_valid
+#                               [3]=rx_empty [4]=rx_valid [5]=rx_full
 #   0x0C  SPI_CTRL    RW [4:0]  [0]=enable [1]=CPOL [2]=CPHA
 #                               [3]=irq_done_en [4]=loopback
 #   0x10  SPI_CLK_DIV RW [15:0] half-period = (DIV+1) clocks
 #   0x14  SPI_CS_CTRL RW [0]    cs_n level (0=assert, 1=deassert)
 
 import cocotb
 from cocotb.clock import Clock
-from cocotb.triggers import RisingEdge, ClockCycles
+from cocotb.triggers import RisingEdge, ClockCycles, FallingEdge
```

Flake8 (7.3.0)

[error] 'cocotb.triggers.FallingEdge' imported but unused

(F401)